boards/nordic/nrf54l20pdk/nrf54l20_cpuapp_common.dtsi

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* This file is common to the secure and non-secure domain */
+
+#include <nordic/nrf54l20_cpuapp.dtsi>
+#include "nrf54l20pdk_nrf54l20-common.dtsi"
+
+/ {
+	chosen {
+		zephyr,console = &uart20;
+		zephyr,shell-uart = &uart20;
+		zephyr,uart-mcumgr = &uart20;
+		zephyr,bt-mon-uart = &uart20;
+		zephyr,bt-c2h-uart = &uart20;
+		zephyr,flash-controller = &rram_controller;
+		zephyr,flash = &cpuapp_rram;
+		zephyr,ieee802154 = &ieee802154;
```

Comment:
I'm curious why `ieee802154` is selected at the board level while `bt-hci` is chosen at the soc level. Should we aim to unify it?

Reply:
I based my code on nRF54L15. It can be question to someone else